app/controllers/statistics_controller.rb

```diff
@@ -1,23 +1,40 @@
 class StatisticsController < ApplicationController
   def index
-    @os_count    = Host.authorized(:view_hosts, Host).count_distribution :operatingsystem
-    @arch_count  = Host.authorized(:view_hosts, Host).count_distribution :architecture
-    @env_count   = Host.authorized(:view_hosts, Host).count_distribution :environment
-    @klass_count = Puppetclass.authorized(:view_puppetclasses).where('total_hosts>0').map{|pc| {:label=>pc.to_label, :data =>pc.total_hosts}}
-    @cpu_count   = FactValue.authorized(:view_facts).my_facts.count_each "processorcount", :unit => Nn_('%s core', '%s cores')
+    @os_count = Host.authorized(:view_hosts, Host).count_distribution :operatingsystem
+    @arch_count = Host.authorized(:view_hosts, Host).count_distribution :architecture
+    @env_count = Host.authorized(:view_hosts, Host).count_distribution :environment
+    @klass_count = Puppetclass.authorized(:view_puppetclasses).where('total_hosts>0').map { |pc| {:label => pc.to_label, :data => pc.total_hosts} }
```

Comment:
nitpick: it looks like a lot of whitespace only changes? if thats the case, I would not include it in this commit.